configs/terraform/environments/prod/image-syncer.tf

```diff
@@ -11,8 +11,8 @@ resource "google_service_account_iam_member" "image_syncer_reader_workflow_sa_us
 
 resource "google_artifact_registry_repository_iam_member" "image_syncer_prod_repo_writer" {
   provider   = google.kyma_project
-  location   = google_artifact_registry_repository.prod_docker_repository.location
-  repository = google_artifact_registry_repository.prod_docker_repository.name
+  location   = module.prod_docker_registry.artifact_registry_collection.location
```

Comment:
Can we get rid of `_collection` suffix from output?

Reply:
We can, but that require changes in all usage of the module. Definitely too large change for that PR.